project/plugins.sbt

```diff
@@ -4,3 +4,4 @@ addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.10.0")
 addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.9.2")
 addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat" % "0.1.20")
 addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.12.0")
+addSbtPlugin("com.eed3si9n" % "sbt-projectmatrix" % "0.8.0")
```

Comment:
Not sure how I feel about this... Do we actually gain anything with this plugin?

Reply:
I see - this probably makes it easier to disable some subprojects. Hopefully CI likes it :)